Maestro/subscriptions.json

```diff
@@ -853,6 +859,36 @@
         "GithubRepo": "<trigger-repo>",
         "BranchToMirror": "<trigger-branch>"
       }
+    },
+    // Orchestrated build final package publish
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/master/build.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/build.semaphore"
+      ],
+      "action": "DotNet-Orchestration-Final-Publish-Executor",
+      "actionArguments": {
+        "vsoBuildParameters": {
+          "PB_FinalTargets": "/t:FinalPackagePublish",
+          "PB_TriggerPath": "<trigger-path>",
+          "PB_VersionsRepoRef": "<trigger-commit>"
+        }
+      }
+    },
+    // Orchestrated build final blob publish
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/master/build.semaphore",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/build.semaphore"
+      ],
+      "action": "DotNet-Orchestration-Final-Publish-Executor",
+      "actionArguments": {
+        "vsoBuildParameters": {
+          "PB_FinalTargets": "/t:FinalBlobPublish",
```

Comment:
Why 2 subscriptions? Can't the one do both targets? or is this to try and do both in parallel?

Reply:
Yep, this is to do them in parallel. Packages and blobs are exclusive, so it's a pretty good place to split it.

Reply:
OK do you expect to add another one for symbols?

Reply:
Yeah, I've (zenhub) attached issues to this epic about the planned steps to add: https://github.com/dotnet/core-eng/issues/2034